src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -123,6 +137,10 @@
 	<meta property="product:price:amount" content={String(data.product.price.amount)} />
 	<meta property="product:price:currency" content={data.product.price.currency} />
 	<meta property="og:type" content="og:product" />
+	{#if data.product.actionSettings.googleShopping.visible}
+		<!-- eslint-disable svelte/no-at-html-tags -->
+		{@html serializeSchema(schema)}
```

Comment:
```suggestion
		<script type="application/ld+json">
			{
				JSON.stringify(schema);
			}
		</script>
```

Did you try this? I suggested to try `@html` only if it didn't work (I even bolded the **if** :) )

Reply:
as i already said, if i have two <script> tag i will not be able to access the variable in the first tag. I tried it but it doesn't work

Reply:
ok :)